samples/python/src/getting_started.py

```diff
@@ -8,7 +8,7 @@
 
 from indy.error import ErrorCode, IndyError
 
-from src.utils import get_pool_genesis_txn_path, run_coroutine
+from utils import get_pool_genesis_txn_path, run_coroutine
```

Comment:
Having absolute imports is better. The problem is a missing `__init__.py` file in `src`. `src` should be a package